tests/test_pond_rd_wr.py

```diff
@@ -39,42 +39,43 @@ def test_pond(data_width=16,  # CGRA Params
                                   check_flip_flop_always_ff=False)
 
     tester = fault.Tester(magma_dut, magma_dut.clk)
-    ###
-    # Ranges, Strides, Dimensionality, Starting Addr, Starting Addr - Schedule
-    ctrl_rd = [[16, 1], [1, 1], 2, 0, 16]
-    ctrl_wr = [[16, 1], [1, 1], 2, 0, 0]
-    pond_config = generate_pond_api(ctrl_rd, ctrl_wr)
+    # Parameterize on start addr for read
```

Comment:
Can you use the pytest parameterize to do this?

Here's an example - https://github.com/StanfordAHA/lake/blob/master/tests/test_reg_fifo.py#L12